test/e2e/fixtures.go

```diff
@@ -414,14 +415,16 @@ func createTestBusyboxPods(tb testing.TB, data *TestData, num int, nodeName stri
 
 	createPodAndGetIP := func() (string, *PodIPs, error) {
 		podName := randName("test-pod-")
-
+		if ns == "" {
```

Comment:
2 separate suggestions:
* move `ns = testNamespace` out of `createPodAndGetIP`
* remove code for `ns = testNamespace` and put it in `podWaitForIPs` instead. No strong preference. Just think that it makes `createBusyboxPodOnNode` and `podWaitForIPs` consistent.

Reply:
Thanks. Followed the 2nd suggestion. I think it makes sense that `podWaitForIPs` also need to handle the case with `"" ` being the namespace input.